Fixed entries in code splitting guide

awkay opened the pull request:
After beating my head against the wall, I read the source...seems like this changed from symbols to strings?

swannodette commented:
@awkay symbols or string should work fine, that's what we actually test with. We also have a bunch of tests case where this is mixed. Can you provide a minimal failing case? Thanks!

awkay commented:
Well, that is embarrassing.  It works when I switch it back. Sorry for the noise. I think I had `:main` set or something last night, and corrected that somewhere along the way. In the interim though I switched to quotes...possibly I made both changes at once and thought the quotes were the fix. The error I was getting was about a PersistentHashSet, and that's the only thing in the config that uses a set.
